host/examples/tx_samples_from_file.cpp

```diff
@@ -201,7 +207,7 @@ int UHD_SAFE_MAIN(int argc, char* argv[])
     }
 
     // set sigint if user wants to receive
-    if (repeat) {
+    if (repeat and (n_repeats > 0)) {
```

Comment:
This should be a or as both options shall not be used together

Reply:
fixed